app/controllers/api/v1/application_controller.rb

```diff
@@ -12,6 +12,7 @@ def current_user
       end
 
       # Render Error Message in json_api format
+      # :reek:LongParameterList { max_params: 5 }
       def render_error(detail:, source: nil, meta: nil, status: :unprocessable_entity, code: nil)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/a52a836dd21ed47bf1fd4740a7355723c867b07a/app/controllers/api/v1/application_controller.rb#L16"></span>Remove unused methods (api::v1::applicationcontroller#render_error)</td>
    </tr>
  </tbody>
</table>

Reply:
We use it in child controllers